pkg/client/dtclient/cache.go

```diff
@@ -0,0 +1,65 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dtclient
+
+import "sync"
+
+type cache[T any] struct {
+	cachedItems map[string][]T
```

Comment:
map-key is what? Schema ID? Please specify or add brief docs.

Reply:
it's **a** key.
might be schema id, depending on the use case...